docs/docs/community/run-locally.md

```diff
@@ -65,7 +65,6 @@ However, if you want to test certain parts of Novu or run it in production mode,
 - `REDIS_DB_INDEX`<br />The Redis database index
 - `REDIS_CACHE_SERVICE_HOST`<br />The domain / IP of your redis instance for caching
 - `REDIS_CACHE_SERVICE_PORT`<br />The port of your redis instance for caching
-- `REDIS_DB_INDEX`<br />The Redis cache database index
```

Comment:
This needs to be added but the name was wrong. Should be `REDIS_CACHE_DB_INDEX`.

Reply:
I had a guess it might also be the case, let me change it real quick 💯